src/context/SyntheticsStateContext/hooks/tradeboxHooks.ts

```diff
@@ -56,3 +75,26 @@ export const useTradeboxNextPositionValues = () => useSelector(selectTradeboxNex
 export const useTradeboxNextLeverageWithoutPnl = () => useSelector(selectTradeboxNextLeverageWithoutPnl);
 export const useTradeboxChooseSuitableMarket = () => useSelector(selectTradeboxChooseSuitableMarket);
 export const useTradeboxGetMaxLongShortLiquidityPool = () => useSelector(selectTradeboxGetMaxLongShortLiquidityPool);
+
+export const useTradeboxSetAdvancedOptions = () => useSelector(selectTradeboxSetAdvancedOptions);
+export const useTradeboxMaxLiquidityPath = () => useSelector(selectTradeboxMaxLiquidityPath);
+export const useTradeboxLiquidity = () => useSelector(selectTradeboxLiquidity);
+export const useTradeboxIsWrapOrUnwrap = () => useSelector(selectTradeboxIsWrapOrUnwrap);
+export const useTradeboxExecutionFee = () => useSelector(selectTradeboxExecutionFee);
+export const useTradeboxAdvancedOptions = () => useSelector(selectTradeboxAdvancedOptions);
+export const useTradeboxMarkPrice = () => useSelector(selectTradeboxMarkPrice);
+export const useTradeboxTriggerPrice = () => useSelector(selectTradeboxTriggerPrice);
```

Comment:
recently discussed with @midas-myth it's not really necessary to create hooks to wrap useSelector,
you can just `useSelector(selectTradeboxTriggerPrice);` where you need it, to avoid these duplicates

Reply:
Removed this file entirely